### ISSUE TITLE
[a11y-media-player] disable interactive transcript while in "learningMode"

### ISSUE THREAD

**Issue by btopro** [elmsln/issues#667]
#55 's PR added support into HAX to make it possible to disable some buttons under the guise of not allowing people to skip around through material when in the context of a quiz (for example). This should also ensure that if we have closed captions for a11y purposes, that we don't expose the interactive transcript which could be used to skip ahead / brick the purpose of the instruction by effectively searching for the word in the question sorta thing. A rare context but also could help w/ the whole branching path questions concept previously mentioned in 55.

**Comment by btopro** [comment]
@Griffin-Sullivan this one for the follow up

**Comment by Griffin-Sullivan** [comment]
Do the closed captions use the transcript in the media player? And is there a way to access the transcript if the captions are on or something?

I can pick this up and try to get it done before Tuesday, but if not I'll finish it afterwards.

**Comment by btopro** [comment]
Transcript is generated by parsing the `.vtt` closed caption file. Those for accessibility reasons cannot be disabled. It provides the black bar, white text over top of the video while someone is speaking. The interactive scrollable transcripts are for all audiences to benefit from, though there is no legal ramification for it not being provided (especially in this context). It's also something that if requested, we'd provide manually (the full text of the video / audio clip for example).

https://webcomponents.psu.edu/styleguide/?path=/story/media-video--a-11-y-media-player-story shows that there are booleans for `hideTranscript` (probably the setting you want) `disableInteractive` `hideTimestamps` and `disableSearch`. My guess is that the `hideTranscript` value just needs bridged from the `video-player` into the `a11y-media-player` just like you did for the `learningMode` setting
